com.gamestarmechanic.app.yaml

```diff
@@ -0,0 +1,77 @@
+---
+app-id: com.gamestarmechanic.app
```

Comment:
it should be `com.gamestarmechanic.gamestarmechanic`. `app` isn't an application name.

Reply:
Oh, sure, that makes sense to me! Should I make a new PR with a different source branch (and these other changes)?

Reply:
branch names are irrelevant